HW0/P6/P6B.py

```diff
@@ -1,11 +1,24 @@
 import multiprocessing as mp
 import time
 import matplotlib.pyplot as plt
+import numbers
+import numpy as np
 
 # Sleep for t seconds
 def burnTime(t):
     time.sleep(t)
 
+# serial run
+def serialRun(t):
+    for i in range(1,16):
+        burnTime(t)
+
+# parallel run
+def parallelRun(t):
+    assert isinstance(t, numbers.Number), "t must be a number"
```

Comment:
An additional check is definitely a good idea. The reason I did it was, that for the serialRun the t has not really a meaning, it is simply passed through. Thus, I just assume that burnTime checks it in order to prevent repeated checking.